### DISCUSSION TITLE
Winning effect name

### DISCUSSION THREAD

**Discussion by Bader-Idris** [cssbattle/forum#349] [Q&A] [answered]
What's the name of the winning effect, and is it pure CSS, or with JS

**Comment by chinchang** [comment]
which one? you talking about getting a 100% match or learn course?

**Reply by Bader-Idris** [reply]
when a competitor wins any challenge on the website. if 100% correct or if even less in some decimals

**Reply by chinchang** [reply, marked as answer]
this -> https://www.npmjs.com/package/canvas-confetti
